R/admin.R

```diff
@@ -2,15 +2,10 @@
 #'
 #' Note that this function can only be used with admin rights.
 #'
-#' @param action The action: add, modify, or remove.
-#' @param target The subject of the action: user, zone, resource, childtoresc,
-#   childfromresc, token, group, rebalance, unusedAVUs, specificQuery.
-#' @param arg2 arg2
-#' @param arg3 arg3
-#' @param arg4 arg4
-#' @param arg5 arg5
-#' @param arg6 arg6
-#' @param arg7 arg7
+#' @param name Name of user to be added.
+#' @param password Password of to be added.
+#' @param action The action: create_user, remove_user, or set_password.
+#' @param role Role of user: rodsuser or rodsadmin.
```

Comment:
could also be ... 'groupadmin'?

Reply:
yes, will change that. Although this function is not exposed to the user, as we opted for user facing functionality only.